packages/helpers/classes/response-error.js

```diff
@@ -40,10 +40,10 @@ class ResponseError extends Error {
    * Convert to string
    */
   toString() {
-    const {body} = this.response;
+    const {data} = this.response;
```

Comment:
This should be reverted back to `body`.

Reply:
This uses the raw response object from `axios`

Reply:
No, it's using the response object defined in L27.